### DISCUSSION TITLE
Table view export

### DISCUSSION THREAD

**Discussion by Scurvy6087** [marcusolsson/obsidian-projects#489] [Help] [answered]
Hello!  First, let me start by saying this plugin is awesome, and love all the hard work you have been putting into it!  I use it to manage my work and personal projects.  Something I frequently have to do is send status update emails which include the project activities. This is the table view in the projects plugin.  Is there a way to export that table into a CSV, excel, or even markdown table? My workflow would be so much faster than having to manage two separate lists. 

Thank you in advance for any help you can provide!

**Comment by marcusolsson** [comment, marked as answer]
That's a really cool idea! Would you mind creating a [feature request](https://github.com/marcusolsson/obsidian-projects/issues/new/choose) for it?

The tricky thing is that to save files outside of the vault we'd need Node.js, which isn't available on mobile. We could export CSV to a folder within the vault (which would work on mobile too), but it would likely be confusing because Obsidian doesn't show unrecognized files in the File explorer.

Alternatively, it could be a desktop-only feature. Or, we could have a "Copy as CSV" which would put the table in the clipboard. Though for really big projects, it's likely not gonna work :)

Feel free to add your thoughts on how you'd want to use it in your feature request. 🙏
